packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -453,6 +474,9 @@ export class ConnectionManager implements IConnectionManager {
             if (this.closed) {
                 throw new Error("Attempting to connect a closed DeltaManager");
             }
+            if (abortSignal.aborted === true) {
+                return;
```

Comment:
Should we log here as well as in the cancelConnection function?  I think that would be good, it's hard to track through this loop in telemetry.  You could include # of attempts, duration, etc - look at the calls to logNetworkFailure below for some ideas.

Reply:
What is the best practice for logging additional information in telemetry events that are not considered errors? Can I just include an "error" object in `this.logger.sendTelemetryEvent()` or is there a better way?

Reply:
Are you looking for `this.logger.sendTelemetryEvent({eventName: "foo", prop1: "bar". prop2: 42 });`?  If you do have an error object you can always pass it as well, even for non-error log events.